### ISSUE TITLE
【EESS】New submissions for Wed, 27 Dec 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#1033]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
### Sparse-view CT Reconstruction with 3D Gaussian Volumetric Representation
 - **Authors:** Yingtai Li, Xueming Fu, Shang Zhao, Ruiyang Jin, S. Kevin Zhou
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2312.15676
 - **Pdf link:** https://arxiv.org/pdf/2312.15676
 - **Abstract**
 Sparse-view CT is a promising strategy for reducing the radiation dose of traditional CT scans, but reconstructing high-quality images from incomplete and noisy data is challenging. Recently, 3D Gaussian has been applied to model complex natural scenes, demonstrating fast convergence and better rendering of novel views compared to implicit neural representations (INRs). Taking inspiration from the successful application of 3D Gaussians in natural scene modeling and novel view synthesis, we investigate their potential for sparse-view CT reconstruction. We leverage prior information from the filtered-backprojection reconstructed image to initialize the Gaussians; and update their parameters via comparing difference in the projection space. Performance is further enhanced by adaptive density control. Compared to INRs, 3D Gaussians benefit more from prior information to explicitly bypass learning in void spaces and allocate the capacity efficiently, accelerating convergence. 3D Gaussians also efficiently learn high-frequency details. Trained in a self-supervised manner, 3D Gaussians avoid the need for large-scale paired data. Our experiments on the AAPM-Mayo dataset demonstrate that 3D Gaussians can provide superior performance compared to INR-based methods. This work is in progress, and the code will be publicly available.
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
### Frequency-Domain Identification of Discrete-Time Systems using  Sum-of-Rational Optimization
 - **Authors:** Mohamed Abdalmoaty, Jared Miller, Mingzhou Yin, Roy S. Smith
 - **Subjects:** Systems and Control (eess.SY); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2312.15722
 - **Pdf link:** https://arxiv.org/pdf/2312.15722
 - **Abstract**
 We propose a computationally tractable method for the identification of stable canonical discrete-time rational transfer function models, using frequency domain data. The problem is formulated as a global non-convex optimization problem whose objective function is the sum of weighted squared residuals at each observed frequency datapoint. Stability is enforced using a polynomial matrix inequality constraint. The problem is solved globally by a moment-sum-of-squares hierarchy of semidefinite programs through a framework for sum-of-rational-functions optimization. Convergence of the moment-sum-of-squares program is guaranteed as the bound on the degree of the sum-of-squares polynomials approaches infinity. The performance of the proposed method is demonstrated using numerical simulation examples.
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### CSI Measurements and Initial Results for Massive MIMO to UAV  Communication
 - **Authors:** Zhuangzhuang Cui, Achiel Colpaert, Sofie Pollin
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2312.15188
 - **Pdf link:** https://arxiv.org/pdf/2312.15188
 - **Abstract**
 Non-Terrestrial Network (NTN) has been envisioned as a key component of the sixth-generation (6G) mobile communication system. Meanwhile, unmanned aerial vehicles (UAVs) play an important role in enabling and deploying NTNs. In this paper, we focus on massive multi-input multi-output (MaMIMO) supported UAV communications, where channel state information (CSI) was measured considering different heights and trajectories of a rotary-wing drone. To characterize the propagation channel for this air-to-ground link, some initial results were analyzed, such as stationary distance. To investigate the impact of channels on communication performance, we analyzed spectral efficiency (SE) by using Maximum Ratio Combining (MRC). This study shows that the presented space-time-frequency channel dataset facilitates channel correlation analysis and supports performance evaluation for MaMIMO-UAV communications.
### UAV Trajectory Tracking via RNN-enhanced IMM-KF with ADS-B Data
 - **Authors:** Yian Zhu, Ziye Jia, Qihui Wu, Chao Dong, Zirui Zhuang, Huiling Hu, Qi Cai
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2312.15721
 - **Pdf link:** https://arxiv.org/pdf/2312.15721
 - **Abstract**
 With the increasing use of autonomous unmanned aerial vehicles (UAVs), it is critical to ensure that they are continuously tracked and controlled, especially when UAVs operate beyond the communication range of ground stations (GSs). Conventional surveillance methods for UAVs, such as satellite communications, ground mobile networks and radars are subject to high costs and latency. The automatic dependent surveillance-broadcast (ADS-B) emerges as a promising method to monitor UAVs, due to the advantages of real-time capabilities, easy deployment and affordable cost. Therefore, we employ the ADS-B for UAV trajectory tracking in this work. However, the inherent noise in the transmitted data poses an obstacle for precisely tracking UAVs. Hence, we propose the algorithm of recurrent neural network-enhanced interacting multiple model-Kalman filter (RNN-enhanced IMM-KF) for UAV trajectory filtering. Specifically, the algorithm utilizes the RNN to capture the maneuvering behavior of UAVs and the noise level in the ADS-B data. Moreover, accurate UAV tracking is achieved by adaptively adjusting the process noise matrix and observation noise matrix of IMM-KF with the assistance of the RNN. The proposed algorithm can facilitate GSs to make timely decisions during trajectory deviations of UAVs and improve the airspace safety. Finally, via comprehensive simulations, the total root mean square error of the proposed algorithm decreases by 28.56%, compared to the traditional IMM-KF.
### Mesure simultan{é}e des r{é}ponses impulsionnelles en  macrodiversit{é}
 - **Authors:** Karim Zayana (M.E.N.E.S.R., COMELEC), Daniel Duponteil
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2312.15952
 - **Pdf link:** https://arxiv.org/pdf/2312.15952
 - **Abstract**
 Macroscopic diversity in radiomobile systems is a technique in which the terminal is in parallel connection with two base stations. It is a fundamental element in the operation of third generation mobile networks using CDMA. (IS 95, UMTS, etc.). A representation of the propagation phenomena involved requires joint modeling of the two transmission channels. The article presents a very simple method for simultaneously measuring their impulse responses, developed for this purpose. This method was the subject of a patent. It can more generally be applied to any number of channels and should allow the reuse of already existing single-channel sounders.
### Measurements and modelisation of shadowing cross-correlations between  two base-stations
 - **Authors:** Karim Zayana (M.E.N.E.S.R., COMELEC), Bertrand Guisnet
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2312.15953
 - **Pdf link:** https://arxiv.org/pdf/2312.15953
 - **Abstract**
 Measurements in macrodiversity situations have been performed. Crosscorrelation of shadowing terms between two basestations and a mobile have been computed. It results that high cross-correlation coefficients may appear in practice. Afterwards, we propose a simple shadowing model, integrating both its autocorrelation and cross-correlation properties. Eventually, this model is used to compute simulations in some typical network cases.
### BSRAW: Improving Blind RAW Image Super-Resolution
 - **Authors:** Marcos V. Conde, Florin Vasluianu, Radu Timofte
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2312.15487
 - **Pdf link:** https://arxiv.org/pdf/2312.15487
 - **Abstract**
 In smartphones and compact cameras, the Image Signal Processor (ISP) transforms the RAW sensor image into a human-readable sRGB image. Most popular super-resolution methods depart from a sRGB image and upscale it further, improving its quality. However, modeling the degradations in the sRGB domain is complicated because of the non-linear ISP transformations. Despite this known issue, only a few methods work directly with RAW images and tackle real-world sensor degradations. We tackle blind image super-resolution in the RAW domain. We design a realistic degradation pipeline tailored specifically for training models with raw sensor data. Our approach considers sensor noise, defocus, exposure, and other common issues. Our BSRAW models trained with our pipeline can upscale real-scene RAW images and improve their quality. As part of this effort, we also present a new DSLM dataset and benchmark for this task.
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Sample selection with noise rate estimation in noise learning of medical  image analysis
 - **Authors:** Maolin Li, Giacomo Tarroni, Vasilis Siomos
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2312.15233
 - **Pdf link:** https://arxiv.org/pdf/2312.15233
 - **Abstract**
 Deep learning techniques have demonstrated remarkable success in the field of medical image analysis. However, the existence of label noise within data significantly hampers its performance. In this paper, we introduce a novel noise-robust learning method which integrates noise rate estimation into sample selection approaches for handling noisy datasets. We first estimate the noise rate of a dataset with Linear Regression based on the distribution of loss values. Then, potentially noisy samples are excluded based on this estimated noise rate, and sparse regularization is further employed to improve the robustness of our deep learning model. Our proposed method is evaluated on five benchmark medical image classification datasets, including two datasets featuring 3D medical images. Experiments show that our method outperforms other existing noise-robust learning methods, especially when noise rate is very big.
### Perception-Distortion Balanced Super-Resolution: A Multi-Objective  Optimization Perspective
 - **Authors:** Lingchen Sun, Jie Liang, Shuaizheng Liu, Hongwei Yong, Lei Zhang
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2312.15408
 - **Pdf link:** https://arxiv.org/pdf/2312.15408
 - **Abstract**
 High perceptual quality and low distortion degree are two important goals in image restoration tasks such as super-resolution (SR). Most of the existing SR methods aim to achieve these goals by minimizing the corresponding yet conflicting losses, such as the $\ell_1$ loss and the adversarial loss. Unfortunately, the commonly used gradient-based optimizers, such as Adam, are hard to balance these objectives due to the opposite gradient decent directions of the contradictory losses. In this paper, we formulate the perception-distortion trade-off in SR as a multi-objective optimization problem and develop a new optimizer by integrating the gradient-free evolutionary algorithm (EA) with gradient-based Adam, where EA and Adam focus on the divergence and convergence of the optimization directions respectively. As a result, a population of optimal models with different perception-distortion preferences is obtained. We then design a fusion network to merge these models into a single stronger one for an effective perception-distortion trade-off. Experiments demonstrate that with the same backbone network, the perception-distortion balanced SR model trained by our method can achieve better perceptual quality than its competitors while attaining better reconstruction fidelity. Codes and models can be found at https://github.com/csslc/EA-Adam.
### Improving the Accuracy and Interpretability of Neural Networks for Wind  Power Forecasting
 - **Authors:** Wenlong Liao, Fernando Porte-Agel, Jiannong Fang, Birgitte Bak-Jensen, Zhe Yang, Gonghao Zhang
 - **Subjects:** Systems and Control (eess.SY); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2312.15741
 - **Pdf link:** https://arxiv.org/pdf/2312.15741
 - **Abstract**
 Deep neural networks (DNNs) are receiving increasing attention in wind power forecasting due to their ability to effectively capture complex patterns in wind data. However, their forecasted errors are severely limited by the local optimal weight issue in optimization algorithms, and their forecasted behavior also lacks interpretability. To address these two challenges, this paper firstly proposes simple but effective triple optimization strategies (TriOpts) to accelerate the training process and improve the model performance of DNNs in wind power forecasting. Then, permutation feature importance (PFI) and local interpretable model-agnostic explanation (LIME) techniques are innovatively presented to interpret forecasted behaviors of DNNs, from global and instance perspectives. Simulation results show that the proposed TriOpts not only drastically improve the model generalization of DNNs for both the deterministic and probabilistic wind power forecasting, but also accelerate the training process. Besides, the proposed PFI and LIME techniques can accurately estimate the contribution of each feature to wind power forecasting, which helps to construct feature engineering and understand how to obtain forecasted values for a given sample.
